pywbem/cim_operations.py

```diff
@@ -4326,14 +4326,21 @@ def IterEnumerateInstances(self, ClassName, namespace=None,
             instances, as follows:
 
             * If `False`, inherited properties are not excluded.
-            * If `True`, inherited properties are basically excluded, but the
+            * If `True`, the behavior depends: If pull operations are used,
+              inherited properties are not excluded. If traditional operations
+              are used, inherited properties are basically excluded, but the
               behavior may be WBEM server specific.
-            * If `None`, this parameter is not passed to the WBEM server, and
+            * If `None`, the behavior depends: If pull operations are used,
```

Comment:
Suggest change "depends" to "depends on underlying operation"

Reply:
Done